docs/reference/ml/trained-models/apis/start-trained-model-deployment.asciidoc

```diff
@@ -55,6 +55,11 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=model-id]
 [[start-trained-model-deployment-query-params]]
 == {api-query-parms-title}
 
+`cache_size`::
+(Optional, <<byte-units,byte value>>)
+The inference cache size (off JVM heap) per node for the model. The default value is the same size as the model
```

Comment:
Drive-by suggestions: "off ... heap" is unclear to me. If it's part of the heap, "in" seems better to me:
```suggestion
The inference cache size (in the JVM heap) per node for the model. The default value is the same size as the model
```
```suggestion
The inference cache size (off JVM heap) per node for the model. The default value is the same size as the model
```

Reply:
I mean "outside" the jvm heap, not part of the heap

Reply:
I think we need to explain that this comes out of ML native memory (using whatever terminology we use for that elsewhere), but also explain that values up to the on-disk size of the model are effectively free.

Reply:
I will look at our docs @droberts195 and update this line